### PR TITLE
Typography-conclusion

### PR DESCRIPTION
**Pull request by Kevin-Solomon**
Just made a typography and conclusion page.
Please let me know your feedback.

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -93,6 +95,10 @@ ul {
 a {
   text-decoration: none;
   color: inherit;
+  cursor: pointer;
+}
+.active {
+  color: var(--action-color);
 }
 .btn {
   padding: 0.5em;
```
Comment (mohammed-sas) [inline index.css:104]: i would suggest you to use "rems" throughtout

File: pages/typography/typography.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            <p class='x-large-text underline'>This is extra large text</p>
+            <p class='large-text underline'>This is large text</p>
+            <p class='medium-text underline'>This is medium text</p>
+            <p class='small-text underline'>This is small text</p>
+            <p class='x-small-text '>This is extra small text</p>
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=1000&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cp%2520class%253D%27x-large-text%27%253EThis%2520is%2520extra%2520large%2520text%253C%252Fp%253E%250A%253Cp%2520class%253D%27large-text%27%253EThis%2520is%2520large%2520text%253C%252Fp%253E%250A%253Cp%2520class%253D%27medium-text%27%253EThis%2520is%2520medium%2520text%253C%252Fp%253E%250A%253Cp%2520class%253D%27small-text%27%253EThis%2520is%2520small%2520text%253C%252Fp%253E%250A%253Cp%2520class%253D%27x-small-text%27%253EThis%2520is%2520extra%2520small%2520text%253C%252Fp%253E"
+              style=" height: 279px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (mohammed-sas) [inline pages/typography/typography.html:76]: avoid inline styling for iframe

**Review by mohammed-sas — CHANGES_REQUESTED** [review]
other than suggested , everything is good to go

**Comment by Husain01** [comment]
Looks good. Nice style guide

File: index.css (hunk 1 above)
Comment (Kevin-Solomon) [reply index.css:104]: I only used ems here to scale with the font size as em when used for lengths depend on the font size

> i would suggest you to use "rems" throughtout

File: pages/typography/typography.html (hunk 2 above)
Comment (Kevin-Solomon) [reply pages/typography/typography.html:76]: > avoid inline styling for iframe

yea this was the default inline style provided by carbonsh

**Comment by Kevin-Solomon** [comment]
> Looks good. Nice style guide

thanks husain

**Pull request merged**